tests/test_exploration_resume.py

```diff
@@ -59,6 +59,51 @@ def test_exploration_in_steps():
     assert exploration.history["gen_informed"][-1]
 
 
+def test_exploration_in_steps_without_limit():
+    """
+    Test that an exploration runs correctly when doing so in several steps
+    without a limit on the maximum number of evaluations.
+    """
+    # Define variables and objectives.
+    var1 = VaryingParameter("x0", -50.0, 5.0)
+    var2 = VaryingParameter("x1", -5.0, 15.0)
+    obj = Objective("f", minimize=False)
+
+    # Define variables and objectives.
```

Comment:
Could you check whether this is correct?
```suggestion
    # Define evaluator.
```

Reply:
good eye!